patroni/postgresql.py

```diff
@@ -1179,7 +1189,17 @@ def primary_conninfo(self, member):
             return None
         r = member.conn_kwargs(self._replication)
         r.update({'application_name': self.name, 'sslmode': 'prefer', 'sslcompression': '1'})
-        keywords = 'user password host port sslmode sslcompression application_name'.split()
+        keywords = 'host port sslmode sslcompression application_name'.split()
```

Comment:
I don't like the ordering change of parameters. We also have to specify the username if it is present.
Basically I would suggest doing something like:
```python
        r.update(application_name=self.name, sslmode='prefer', sslcompression='1', krbsrvname=self._krbsrvname)
        keywords = 'user password host port sslmode sslcompression application_name krbsrvname'.split()
        return ' '.join('{0}={{{0}}}'.format(kw) for kw in keywords if r.get(kw)).format(**r)
```